ccd-definition/CaseField/CareSupervision/messageJudge.json

```diff
@@ -83,25 +92,6 @@
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "hasJudicialMessages",
```

Comment:
take it these don't get persisted (so don't need to be deprecated)?

Reply:
Yes, these are temporary fields used within the message flow and should be deleted at the end of the event.
But... maybe I should move it to deprecated in case they were left over by any reason.